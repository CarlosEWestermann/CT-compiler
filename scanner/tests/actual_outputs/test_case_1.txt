syntax error, unexpected TK_LIT_INT, expecting end of file