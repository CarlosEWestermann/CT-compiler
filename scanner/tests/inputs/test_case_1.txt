int x = 5; // Declaração de uma variável
/* Isso é um
comentário multi-linha */
float y = 10.32;
bool isTrue = false;