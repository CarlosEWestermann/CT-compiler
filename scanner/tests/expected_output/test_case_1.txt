1 TK_PR_INT [int]
1 TK_IDENTIFICADOR [x]
1 TK_ESPECIAL [=]
1 TK_LIT_INT [5]
1 TK_ESPECIAL [;]
4 TK_PR_FLOAT [float]
4 TK_IDENTIFICADOR [y]
4 TK_ESPECIAL [=]
4 TK_LIT_FLOAT [10.32]
4 TK_ESPECIAL [;]
5 TK_PR_BOOL [bool]
5 TK_IDENTIFICADOR [isTrue]
5 TK_ESPECIAL [=]
5 TK_LIT_FALSE [false]
5 TK_ESPECIAL [;]
